common/src/main/java/com/microsoft/identity/common/internal/platform/KeyStoreAccessor.java

```diff
@@ -103,8 +103,8 @@ public static IKeyAccessor forAlias(@NonNull final Context context, @NonNull fin
             return getKeyAccessor((IDevicePopManager.Cipher) suite.cipher(), suite.signingAlgorithm(), popManager);
         }
         final KeyStore instance = KeyStore.getInstance(ANDROID_KEYSTORE);
-        final DeviceKeyManager<KeyStore.SecretKeyEntry> keyManager = new DeviceKeyManager<>(instance, alias, symmetricThumbprint(alias, instance));
```

Comment:
do we still use getSecretKeyThumbprint?

Reply:
My understanding is that it's moved into AbstractKeyStoreKeyManager.

Reply:
Yes, moved to AbstractKeyStoreKeyManager. Removed the old method now. Addressed.